src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -73,19 +71,19 @@ def _create_session(self, alias, url, headers, cookies, auth, timeout, max_retri
 
         # verify can be a Boolean or a String
         if isinstance(verify, bool):
-          s.verify = verify
-        elif isinstance(verify, unicode) or isinstance(verify, str):
-          if verify == 'True':
-            s.verify = True
-          elif verify == 'False':
-            s.verify = False
-          else:
-            # a CA bundle path
             s.verify = verify
+        elif isinstance(verify, unicode) or isinstance(verify, str):
+            if verify == 'True':
```

Comment:
Looks current logic is more complex, than just conversion.
Because requests library allows passing path to cert file: http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification